src/meta/src/hummock/hummock_manager.rs

```diff
@@ -497,24 +480,44 @@ where
     }
 
     #[named]
-    pub async fn unpin_version(
+    pub async fn unpin_version_before(
```

Comment:
Would you please add doc for this method?

Reply:
> Would you please add doc for this method?

added